packages/runtime-vapor/src/component.ts

```diff
@@ -20,11 +21,69 @@ export interface ComponentInternalInstance {
 
   component: FunctionalComponent | ObjectComponent
   get isMounted(): boolean
+  get isUnMounted(): boolean
   isMountedRef: Ref<boolean>
+  isUnMountedRef: Ref<boolean>
```

Comment:
@sxzz 
I don't quite understand the handling of this flag.
At the very least, if implementing 'isUnmounted', it would probably be good to rewrite it as true during unmount.

---

ref:


https://github.com/vuejs/core-vapor/pull/42

> Flags related to lifecycle
I had implemented it so that isMounted is set to false when unmounting,
In traditional components, it seems that isUnmounted is set to true.
Which approach should we follow for the implementation? (In this PR, I have added the isUnmounted flag.)

Reply:
The default value of 'isUnmounted' in 'runtime-core' is false, so I have set it to false.

Reply:
Yes, that seems to be the correct default value,
but I do think it should be marked as true when the Component is unmounted.

(However, currently when unmounting, the implementation sets isMounted to false, so I'm not quite clear on how these two flags should be treated...)

Reply:
Oops, I forgot. My bad. 😢

Reply:
Oh no, please don't feel bad. 😸 

In any case, since I don't have a clear grasp of how these two flags should be treated, I'd like to leave the rest to @sxzz ...